packages/amplify_auth_plugin_interface/lib/amplify_auth_plugin_interface.dart

```diff
@@ -16,12 +16,15 @@
 library amplify_auth_plugin_interface;
 
 import 'dart:async';
+import 'package:amplify_auth_plugin_interface/src/Session/AuthUserAttributeRequest.dart';
 import 'package:flutter/foundation.dart';
 import 'package:meta/meta.dart';
 import 'package:amplify_core/types/index.dart';
+import 'src/Session/AuthUserAttribute.dart';
 import 'src/types.dart';
 export 'src/types.dart';
 
+// ignore_for_file: public_member_api_docs
```

Comment:
We probably shouldn't add this to our interface files. All public API methods **should** have documentation.

Reply:
Yeah, good point.